devel/mimalloc/Portfile

```diff
@@ -22,14 +26,27 @@ checksums           rmd160  362b6ba6270889af10af46e41e469847a42b99a0 \
                     sha256  2b1bff6f717f9725c70bf8d79e4786da13de8a270059e4ba0bdd262ae7be46eb \
                     size    1163868
 
-patchfiles          patch-cmake.diff
+patchfiles          patch-cmake.diff \
+                    0001-prim.h-support-tls_slot-for-PPC.patch \
+                    0002-Easy-on-tests-for-PPC.patch
 
 compiler.cxx_standard   2017
 compiler.c_standard     2011
 
 configure.args-append \
                     -DMI_BUILD_TESTS=ON
 
+if {[string match *gcc* ${configure.compiler}]} {
```

Comment:
you think gcc being the compiler causes the issue? That would be unexpected…

Reply:
Clang won’t link with `libatomic`, and GCC is only used by default on PPC.
But you are right, perhaps it makes sense to separate args into separate groups with different conditions.

I have no idea though which OS range should be used. Someone needs to run tests and compare what works better on Intel, I guess.

Reply:
Like the song goes…. “That someone…. is you…” :)

No, really though. It’s time to kick these fixes up a notch, really apply them where they are needed, and make sure they are right.

It’s easy to set up a Leopard i386 VM, many of us have one. You already have SnowLeopard going.

Reply:
Well, I believe in benefits of division of labor :)

Likewise, pretty no one is testing anything on PPC when either submitting new ports, updating existing ones or making fixes, and I do not expect everyone installing 10.5.8 / 10.6.8 Server in a VM to do that testing (not a complicated thing to do though, right?), similarly, I am neither interested nor have time and other resources to maintain another dozen of build systems. Nor see a point – there are plenty of members here who are able to test on Intel, and much better than me. For me it is a waste of time – at least in a sense I rather use the same for something that others won’t do.

Reply:
Well, you've stated your position, at least :) 

Your patches will only ever be tested by you on your custom setups. The rest of MacPorts can just hope they work, or follow along behind you and try to fix them properly if they are broken.

That will, indeed, make them much more difficult for anyone to commit. But at least you're very clear about where you stand.

Reply:
From the very fact that no one committed a fix for Intel systems for this port we can deduce that either a) everything works fine on Intel, and my patches can be restricted to PowerPC (and then the problem you pointed to does not exist), or b) no one cares about older Intel systems and no one ever bothered to test the port on those (and then why should we).
In any case, I address a specific bug – the port is currently broken on PPC systems. If there is any concern about applicability of my patches to Intel, I eagerly make everything PPC-only.

Reply:
@kencu On the other hand, if you are actually concerned about the port in question and not just enjoying an argument, why dont’t you run tests for it? And then we find out if Intel needs any fixes or not :)

Reply:
I doubt this is strictly GCC related too. I think put that condition underneath the os version check as well.